docs-site/package.json

```diff
@@ -17,30 +17,35 @@
     "prettier": "prettier --write ."
   },
   "dependencies": {
-    "@headlessui/react": "^1.6.6",
-    "@heroicons/react": "^1.0.6",
-    "@mdx-js/react": "^2.1.3",
-    "@tailwindcss/typography": "^0.5.4",
+    "@headlessui/react": "^1.7.13",
+    "@heroicons/react": "^2.0.17",
+    "@mdx-js/react": "^2.3.0",
+    "@tailwindcss/typography": "^0.5.9",
     "clsx": "^1.2.1",
-    "gatsby": "^4.21.1",
-    "gatsby-plugin-postcss": "^5.21.0",
-    "gatsby-plugin-sharp": "^4.21.0",
-    "gatsby-plugin-sitemap": "^5.21.0",
-    "gatsby-remark-images": "^6.21.0",
-    "gatsby-source-filesystem": "^4.21.0",
-    "gatsby-transformer-remark": "^5.21.0",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0"
+    "gatsby": "^5.8.1",
+    "gatsby-plugin-postcss": "^6.8.0",
+    "gatsby-plugin-sharp": "^5.8.1",
+    "gatsby-plugin-sitemap": "^6.8.0",
+    "gatsby-remark-images": "^7.8.0",
+    "gatsby-source-filesystem": "^5.8.0",
+    "gatsby-transformer-remark": "^6.8.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@types/node": "^17.0.45",
-    "@types/react": "^18.0.17",
-    "@types/react-dom": "^18.0.6",
-    "autoprefixer": "^10.4.8",
-    "postcss": "^8.4.16",
-    "prettier": "^2.7.1",
-    "prettier-plugin-tailwindcss": "^0.1.13",
-    "tailwindcss": "^3.1.8",
-    "typescript": "^4.7.4"
+    "@types/node": "^18.15.11",
+    "@types/react": "^18.0.34",
+    "@types/react-dom": "^18.0.11",
+    "autoprefixer": "^10.4.14",
+    "postcss": "^8.4.21",
+    "prettier": "^2.8.7",
+    "prettier-plugin-tailwindcss": "^0.2.7",
+    "tailwindcss": "^3.3.1",
+    "typescript": "^5.0.4"
+  },
+  "overrides": {
```

Comment:
This seems a little non-standard  - worth noting why in a comment somewhere?

Reply:
Good call commented ... though comments in JSON 🤮